.github/workflows/zombienet.yml

```diff
@@ -0,0 +1,63 @@
+name: zombienet
+run-name: Zombienet tests
+on:
+  pull_request:
+
+jobs:
+  zombienet-tests:
+    runs-on: ubuntu-latest
```

Comment:
should we run this on the fancy runner we have? 🤔

Reply:
when I changed this to run-on `buildjet-4vcpu-ubuntu-2204`
It give error below when trying to start gossamer.  Any ideas why?

```
zombienet test -p native zombienet_tests/functional/0001-basic-network.zndsl
  shell: /usr/bin/bash -e {0}
┌────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│ 🛎️ Tests are currently running. Results will appear at the end                                                         │
└────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
	 Launching network... this can take a while.
╔════════════════════╤════════════════════════════════════════════════════════════════════════════════════════════════════╗
║ 🧟 Zombienet 🧟    │ Initiation                                                                                         ║
╟────────────────────┼────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ Provider           │ native                                                                                             ║
╟────────────────────┼────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ Namespace          │ zombie-1c79aef4ad1d[3](https://github.com/ChainSafe/gossamer/actions/runs/4601903285/jobs/8130341208#step:9:3)d6f[4](https://github.com/ChainSafe/gossamer/actions/runs/4601903285/jobs/8130341208#step:9:5)04c511a44dbed17                                                            ║
╟────────────────────┼────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ Temp Dir           │ /tmp/zombie-1c79aef4ad1d3d6f404c511a44dbed17_-6320-5uRQMjjnlyYA                                    ║
╚════════════════════╧════════════════════════════════════════════════════════════════════════════════════════════════════╝

		 🚧 Chain Spec was set to a file in raw format, can't customize. 🚧
╔════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════╗
║ Chain name: Westend Local Testnet                                                                                      ║
╚════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════╝
┌────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│ Westend Local Testnet ⚙ Clear Boot Nodes                                                                               │
└────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
┌────────────────────┬────────────────────────────────────────────────────────────────────────────────────────────────────┐
│ Pod                │ alice                                                                                              │
├────────────────────┼────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Status             │ Launching                                                                                          │
├────────────────────┼────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Command            │ gossamer --chain /tmp/zombie-1c79aef4ad1d3d6f404c[5](https://github.com/ChainSafe/gossamer/actions/runs/4601903285/jobs/8130341208#step:9:6)11a44dbed17_-6320-5uRQMjjnlyYA/alic              │
│                    │ e/cfg/westend-local.json --name alice --rpc-cors all --unsafe-rpc-external --rpc-methods u         │
│                    │ nsafe --unsafe-ws-external --key alice --babe-lead --no-mdns --node-key 2bd806c97f0e00af1a         │
│                    │ 1fc3328fa763a9269723c8db8fac4f93af71db186d6e90 --no-telemetry --prometheus-external --vali         │
│                    │ dator --prometheus-port 42641 --rpc-port 41853 --ws-port 39221 --listen-addr /ip4/0.0.0.0/         │
│                    │ tcp/46713/ws --base-path /tmp/zombie-1c79aef4ad1d3d6f404c511a44dbed17_-6320-5uRQMjjnlyYA/a         │
│                    │ lice/data                                                                                          │
└────────────────────┴────────────────────────────────────────────────────────────────────────────────────────────────────┘
uncaughtException
Error [ERR_STREAM_WRITE_AFTER_END]: write after end
    at new NodeError (node:internal/errors:3[7](https://github.com/ChainSafe/gossamer/actions/runs/4601903285/jobs/8130341208#step:9:8)2:5)
    at _write (node:internal/streams/writable:319:11)
    at WriteStream.Writable.write (node:internal/streams/writable:334:10)
    at Socket.ondata (node:internal/streams/readable:754:22)
    at Socket.emit (node:events:527:2[8](https://github.com/ChainSafe/gossamer/actions/runs/4601903285/jobs/8130341208#step:9:9))
    at addChunk (node:internal/streams/readable:315:12)
    at readableAddChunk (node:internal/streams/readable:28[9](https://github.com/ChainSafe/gossamer/actions/runs/4601903285/jobs/8130341208#step:9:10):9)
    at Socket.Readable.push (node:internal/streams/readable:228:[10](https://github.com/ChainSafe/gossamer/actions/runs/4601903285/jobs/8130341208#step:9:11))
    at Pipe.onStreamRead (node:internal/stream_base_commons:[19](https://github.com/ChainSafe/gossamer/actions/runs/4601903285/jobs/8130341208#step:9:20)0:[23](https://github.com/ChainSafe/gossamer/actions/runs/4601903285/jobs/8130341208#step:9:24)) {
  code: 'ERR_STREAM_WRITE_AFTER_END'
}
Error: Process completed with exit code 100.
```

Reply:
Could be related [with this](https://github.com/paritytech/zombienet#what-is-zombienet)?

> Note: Currently, it is only possible to use podman for Zombienet users on Linux machines. Although podman comes with support for macOS, it is done using an internal VM and the Zombienet provider code expects podman to be running natively.

And I think you are using native to run it, did you try with podman provider?

Reply:
No idea, kinda weird. Maybe some race condition in the zombienet code which shows up on faster machines 🤔 Let's keep it to Ubuntu 😄